frontend/src/App.js

```diff
@@ -50,9 +50,15 @@ function App() {
       <Navigation setCurrentView={setCurrentView} />
       <header className="App-header">
         {views[currentView]}
+        <div>
+          <h2>Target Zones</h2>
+          <p>Left Foot: {stepTime.targetZones.left.min} - {stepTimeData.targetZones.left.max}</p>
+          <p>Right Foot: {stepTime.targetZones.right.min} - {stepTimeData.targetZones.right.max}</p>
+          <p>Average Target Zone: {stepTime.targetZones.average}</p>
```

Comment:
![image](https://github.com/user-attachments/assets/cfc94ee5-8c84-4300-8923-a8deefea329a)

Reply:
This error needs to be addressed.
Also, it'd be great to put a null check here to prevent the app from breaking.

Reply:
![Screenshot 2024-10-21 195829](https://github.com/user-attachments/assets/03e92713-5cc4-4a8e-9a7e-89a3ac796f6d)